app/routing/VersionRoutingRequestHandler.scala

```diff
@@ -43,30 +42,45 @@ class VersionRoutingRequestHandler @Inject() (versionRoutingMap: VersionRoutingM
       filters = filters.filters
     ) {
 
-  private val featureSwitch = FeatureSwitches(config.featureSwitches)
-
   private val unsupportedVersionAction = action(Results.NotFound(Json.toJson(UnsupportedVersionError)))
```

Comment:
Need to update this to `private val unsupportedVersionAction = action(Results.NotFound(UnsupportedVersionError.asJson))`